Views/home.stencil

```diff
@@ -13,16 +13,58 @@
     <p>Hack Cambridge</p>
   </section>
   <section class="LandingUpdateFeed">
-    <div class="LandingUpdateFeed__item">
-      <div class="PostCard">
-        <span class="PostCard__label">Workshop</span>
-        <h3 class="PostCard__title">Conquering Linux</h3>
+    <div><h1 class="LandingUpdateFeed__title Text--sectionHeading">Updates</h1></div>
+    <div class="LandingUpdateFeed__postContainer">
+      <div class="LandingUpdateFeed__item">
+        <div class="PostCard">
+          <div class="PostCard__photoBackground" style="background-image: url('/static/images/workshop.jpg')"></div>
+          <div class="PostCard__content">
```

Comment:
This content container is here to lift the content above the photo background. Any ideas for how to ditch this?